### PR TITLE
Update stats for prod81

### PR DESCRIPTION
**Pull request by sbesson**
Adjusts the stats for a few recent releases using https://github.com/IDR/idr-utils/pull/8. Also includes the comments from https://github.com/IDR/idr.openmicroscopy.org/pull/81#discussion_r419266655

### CODE REVIEW

**Comment by dominikl** [comment]
Looks good. Guess I don't really get the terminology in these files because I got quite a few numbers wrong earlier it seems.

File: _data/studies.tsv [hunk 1]
```diff
@@ -75,6 +75,6 @@ idr0066-voigt-mesospim	experimentC	prod70	853	3	0	3		0	4	3739	0.03797700596	3797
 idr0066-voigt-mesospim	experimentD	prod70	854	1	0	1		0	2	2978	0.04118653215	41186532145	6	6864.422024	5025 x 6735 x 1489 x 1 x 1
 idr0067-king-yeastmeiosis	experimentA	prod72	904	60	0	60		0	419	136360	0.2854708875	285470887482	1594	179.0908955	1224 x 1224 x 8 x 2 x 36
 idr0069-caldera-perturbome	screenA	prod71	2251	124	47616	47616			190464	571392	1.225666503	1225666502656	571912	2.143103314	1360 x 1024 x 1 x 3 x 1
-idr0073-schaadt-immuneinfiltrates	experimentA	prod74	1002	193	0	3			193	579	0.02230089	24520087958	580	121.1615447	5578 x 5048 x 1 x 3 x 1
+idr0073-schaadt-immuneinfiltrates	experimentA	prod80	1002	193	0	3			193	579	0.02230089	24520087958	580	121.1615447	5578 x 5048 x 1 x 3 x 1
 idr0075-cabirol-honeybee	experimentA	prod73	951	10	0	10			20	1932	0.0009225002	1014299685	81	48.3655779362	512 x 512 x 100 x 1 x 1
-idr0083-lamers-sarscov2	experimentA	prod80	1051			2		2	2	2	0.067	71359237166	2	33229	111872 x 130304 x 1 x 1 x 1
+idr0083-lamers-sarscov2	experimentA	prod81	1051	1	0	1		0	2	2	0.07	71359237166	2	33229.	111872 x 130304 x 1 x 1 x 1
```
Comment (dominikl) [inline _data/studies.tsv:80]: Just the dot in `33229.` seems to be a bit weird.

**Comment by sbesson** [comment]
Pushed another commit fixing the calculated average sizes.

Happy to have a very stance at clarifying the terminology. What would be the most suitable format? A [datapackage JSON](https://frictionlessdata.io/data-package/) accompanying the TSV files  would be one option.

**Comment by dominikl** [comment]
👍 Good to merge from my point of view. Could give this datapackage json a try... a bit worried though that it's overcomplicating things.

**Comment by sbesson** [comment]
Merging in order to open a follow-up `prod82` stats PR. The discussion around the vocabulary used for IDR metrics can happen separately including the question of how and where to store and exchange it.

**Pull request merged**